decentraland/screens/EthereumSignUpScreen.js

```diff
@@ -21,12 +22,16 @@ export class EthereumSignUpScreen extends React.Component {
       // See more: https://github.com/tasitlabs/tasit/issues/42
       const account = Account.create();
       setAccount(account);
+      showInfo(`Account generated`);
 
       const { address: accountAddress } = account;
 
-      await fundAccount(accountAddress);
-      await approveManaSpending(account);
-      showInfo(`Account created and funded!`);
+      await fundAccountWithEthers(accountAddress);
+      showInfo(`Account funded with ethers`);
+      const fundWithMana = fundAccountWithMana(accountAddress);
+      const approveMarketplace = approveManaSpending(account);
+      await Promise.all([fundWithMana, approveMarketplace]);
+      showInfo(`The account is ready to buy!`);
```

Comment:
"buy" is usually a transitive verb, so it's mostly used like this: "buy `[SOMETHING]`".

Reply:
```suggestion
      showInfo(`Now you can buy land!`);
```